tools/install.pl

```diff
@@ -201,10 +203,14 @@ sub cp_node_module {
     my $nodename = getcwd . "/node_modules" . $item;
     $item =~ /([^\/]+$)/;
     my $newname = getcwd . $newpath . $&;
+    my $nodemapname = $nodename . ".map";
+    my $newmapname = $newname . ".map";
 
-    say("Copying $nodename \r\n to $newname \r\n");
+    say("\r\nCopying $nodename \r\n to $newname");
     copy( $nodename, $newname ) or die "The copy operation failed: $!";
 
+    my $mapresult = copy( $nodemapname, $newmapname ) and say("Copied sourcemap file.\r\n");;
```

Comment:
source maps files will not be loaded into Browser unless user entered the `Developer's Tools`.
It will be fine to leave it, unless you cares about the docker image size very very much.

Reply:
I try to keep the docker image as small as possible so yeah, I'd rather have them out sadly 😅  
If you want to keep them for source builds/debug purposes we could remove them in the cleanup phase of the Docker image build here: https://github.com/Difegue/LANraragi/blob/2cb71c2c601927d6ec79d5e2b459327770efa0d3/tools/build/docker/install-everything.sh#L30

Reply:
Please have a look if the added line works correctly. (Test fine on my machine.)